notebooks/1_Introduction_to_ESMValTool.ipynb

```diff
@@ -0,0 +1,1247 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to ESMValTool\n",
+    "\n",
+    "[ESMValTool](https://github.com/ESMValGroup/ESMValTool) is a library of climate data analysis workflows (\"recipes\") and [ESMValCore](https://github.com/ESMValGroup/ESMValCore) is the tool that can run those recipes. With ESMValTool, it is easy to reproduce already published analyses as well as develop new ones. A useful feature is that you can directly access all the output (data, images, etc) and further process them in the notebook. ESMValTool has specifically been designed to analyse data produced as part of the [Climate Model Intercomparison Project](https://www.wcrp-climate.org/wgcm-cmip/wgcm-cmip6), though it also [supports many observational and reanalysis datasets](https://docs.esmvaltool.org/en/latest/input.html#supported-datasets-for-which-a-cmorizer-script-is-available). It is used extensively to (re)produce the analyses in the [IPCC Assessment Reports](https://www.ipcc.ch/assessment-report/ar6/)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'en_US.UTF-8'"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# This is needed because the preferred encoding on the Levante Jupyter lab server is not set to utf-8.\n",
+    "import locale\n",
+    "locale.setlocale(locale.LC_ALL, 'en_US.UTF-8')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/k/k206100/.conda/envs/esmvaltool/lib/python3.10/site-packages/esmvalcore/experimental/_warnings.py:13: UserWarning: \n",
+      "  Thank you for trying out the new ESMValCore API.\n",
+      "  Note that this API is experimental and may be subject to change.\n",
+      "  More info: https://github.com/ESMValGroup/ESMValCore/issues/498\n",
+      "ERROR 1: PROJ: proj_create_from_database: Open of /home/k/k206100/.conda/envs/esmvaltool/share/proj failed\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Import the tool\n",
+    "import esmvalcore.experimental as esmvaltool\n",
+    "\n",
+    "# Import tools for plotting\n",
+    "import matplotlib.pyplot as plt\n",
+    "import iris.quickplot"
```

Comment:
Yes, if something is only used a few times I prefer to use the full path, so you can immediately see what the function is instead of having to scroll back to the imports at the top of the file to find out.

Reply:
I agree with you in principle, but would argue that there are a couple of exceptions that are so common, that they are standard. This includes `np` for `numpy`, `da` for `dask.array`, `xr` for `xarray`, and most relevant in this context `plt` for `matplotlib.pyplot`. I think it makes sense to follow the Iris documentation in establishing `iplt` for `iris.plot` and `qplt` for `iris.quickplot`. But as I said before, one can certainly differ. If this is the only outstanding thing, feel free to just merge.